package/base-files/files/lib/functions/ipv4.sh

```diff
@@ -0,0 +1,142 @@
+#
+# rewrite of /bin/ipcalc.sh from awk to shell for use as a library
+# in init.d wrappers, and better portability (since busybox awk
+# lacks -E support at this time).
+#
+
+decimal=0
```

Comment:
This thing bugs me.
It has global variable vibes but has an ambiguous name and is lowercase.
The only thing it does is 'neutering' `int2ip` - it's not used anywhere else.

I recommend against making it part of the shell library.
In `ipcalc.sh`, the variable can still be used to forgo int2ip to achieve the same result.

Reply:
Okay, makes it less legible but...